ops/terraform/environments/local/root_variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "region" {
```

Comment:
There's some duplication in the variable definitions and provider declarations that I had trouble working around. Modules expose all input and output variables declared as such to other modules, and it feels like a Fortran COMMON block almost 👴 .